fairroot/alignment/FairAlignmentHandler.cxx

```diff
@@ -100,15 +101,15 @@ void FairAlignmentHandler::AddAlignmentMatrices(const std::map<std::string, TGeo
 
 void FairAlignmentHandler::RecomputePhysicalAssmbBbox() const
 {
-    TObjArray* pPhysNodesArr = gGeoManager->GetListOfPhysicalNodes();
-
-    for (auto pPhysNode : TRangeDynCast<TGeoPhysicalNode>(pPhysNodesArr)) {
-        if (pPhysNode) {
-            auto pShapeAsb = dynamic_cast<TGeoShapeAssembly*>(pPhysNode->GetShape());
-            if (pShapeAsb) {
-                // Should reach here only if the original node was a TGeoShapeAssembly
-                pShapeAsb->ComputeBBox();
-            }
+    TIter nextv(gGeoManager->GetListOfVolumes());
+    TGeoVolume* vol;
+    while ((vol = dynamic_cast<TGeoVolume*>(nextv()))) {
+        if (vol->IsAssembly())
+            vol->GetShape()->ComputeBBox();
```

Comment:
I would prefer braces even around single statements. But that's really personal by now.

```suggestion
        if (vol->IsAssembly()) {
            vol->GetShape()->ComputeBBox();
        }
```

Reply:
Done, I also prefer it so and just applied too quickly the proposal from the CI test.